ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -468,7 +468,7 @@
     "DisplayContext": "COMPLEX",
     "PageID": "Experts",
     "CaseEventFieldLabel": "Experts",
-    "PageShowCondition": "responseClaimTrack= \"FAST_CLAIM\" AND specDefenceFullAdmittedRequired!=\"Yes\"",
+    "PageShowCondition": "responseClaimTrack= \"FAST_CLAIM\" AND respondent1ClaimResponseTypeForSpec!=\"FULL_ADMISSION\"",
```

Comment:
since you have removed specDefenceFullAdmittedRequired!=\"Yes\"", from pageshowcondition, what would happen if specDefenceFullAdmittedRequired=\"No\"" ?

would the page display ?

Reply:
For the Admit and Pay journey, this page should not be shown. Therefore, the page show condition is changed in order to prevent this page show in this journey whenever it is yes or no.